### PR TITLE
[WIP] Prepared conv1d layers for tfjs.

### PR DESCRIPTION
**Pull request by jusjusjus**
I work towards adopting the module to allow for a seemless integration
with `tensoflow.js`.  My usecase targets Conv1d layers, which is why
changes to convert_conv are only partial.  There are two main changes:

* pytorch uses 'NCW' which needs to be converted to 'NWC' for
tensorflow.  However, we want to retain the cool names of our graph's
operators, which is why we need to create a temporary new one that will
hold the ops with changed order.

* PaddingLayers are not supported by tfjs as of now.  We therefore allow
for a 'valid' mode conv1d layer where the padding helper-layer is
omitted.  We also allow to provide the data-format in `params`.

* We need to avoid the long random-number names b/c these aren't
supported by tfjs.

### CODE REVIEW

**Comment by jusjusjus** [comment]
Let me know if you'd be interested in this direction of work, and I'll work towards making the changes consistent across all layers in `pytorch2keras/layers.py` as well as adding some testing.

**Comment by nerox8664** [comment]
Hello @jusjusjus.
Yep, it's very interesting direction. To be honest, this project was created a half of year ago to make some PyTorch models is able to run on Keras.js. I've also tried to make it work on Tensorflow.js, but not so successful.

I've tested your changes and noticed that it breaks support of changing channels order for 4D tensors, like convolution2d.

File: pytorch2keras/converter.py [hunk 1]
```diff
@@ -169,41 +169,50 @@ def pytorch_to_keras(
...
-            model_tf_ordering.layers, model.layers
-        ):
-            dst_layer.set_weights(src_layer.get_weights())
+        # Set the weights into the model with new ordering
+        src_weights = [layer.get_weights() for layer in model.layers]
+        if K.backend() == 'tensorflow':
+            # Tensorflow needs a new graph for the converted model
+            # to retain the same scopes for the operators.
+            import tensorflow as tf
+            with tf.Graph().as_default():
```
Comment (jusjusjus) [inline pytorch2keras/converter.py]: Here I broke your test `conv2d_channels_last.py` because the new model isn't actually stored in the default graph.

I think I fixed both issues.

**Comment by nerox8664** [comment]
hello again @jusjusjus.
I've performed some tests with your changes. There is one more problem with name mangling, you can check it with `resnet18_channels_last.py` or any other resnet.
The problem is - names may be non-unique, it's very important for big networks with nested modules.
